src/audiences/ui/list.js

```diff
@@ -103,13 +103,19 @@ class List extends Component {
 	 */
 	onSearch = event => {
 		const value = event.target.value;
+		const canCreate = this.props.canCreate();
```

Comment:
Not sure how this bit was working but `this.props.canCreate` is the value we passed through in the `applyWithSelect` higher order component. Here it's being called like a function when the value will be a boolean.

I'm suggesting removing this here and just checking `this.props.canCreate` below

```suggestion
```